cypress/e2e/shared/scriptQueryBuilder.sql.test.ts

```diff
@@ -82,8 +95,8 @@ describe('Script Builder', () => {
         confirmSchemaComposition()
 
         cy.log('select field --> adds to composition')
-        const fieldName0 = 'station_id_0'
-        const fieldName10 = 'station_id_10'
+        const fieldName0 = 'air_degrees'
+        const fieldName10 = 'humidity'
```

Comment:
This is small, but why not reuse the const from line 11 and 12 for `air_degrees` and `humidity`?

Reply:
ah, good point. Done.